Makefile

```diff
@@ -164,7 +174,7 @@ $(APPIMAGE_TOOL):
 	mv $(_APPIMAGE_TOOL) tmp/linux/tools/
 	chmod +x $(APPIMAGE_TOOL)
 
-APPIMAGE := pkg/NimStatusClient-x86_64.AppImage
+APPIMAGE ?= pkg/NimStatusClient-x86_64.AppImage
```

Comment:
Are you sure you want to deal with the giant global namespace that is the environment? Without application-specific prefixes uglifying all vars?

Reply:
Yeah, why not, we can rename them `STATUS_CLIENT_APPIMAGE` and `STATUS_CLIENT_DMG`.